### ISSUE TITLE
Delete User in Traccar Web Interface

### ISSUE THREAD

**Issue by SoheylAkhlaghiAmiri** [traccar/traccar#4097]
Hi
I use traccar by SQL Server. Everything okay but when i remove the created user....i faced with an SQLException in traccar web interface.

```
Invalid object name 'user_user'. - SQLServerException (... < QueryBuilder:477 < DataManager:470 < BaseObjectManager:107 < ...)
```

What should i do?

Thanks

**Comment by tananaev** [comment]
What version of Traccar are you using? Is everything default with any modifications to code or config?

**Comment by SoheylAkhlaghiAmiri** [comment]
i use "traccar-windows-4.0"

yes....everything is default. 
I just config to use SQL Server and it works well, this issue is the only problem that i faced.

**Comment by Abyss777** [comment]
Unfortunately I found out that we have  an issue with MS SQL in 4.0.
You need to manually correct table names in triggers `tg_groups_delete` and `tg_users_delete`

Result should be:
```sql
          CREATE TRIGGER tg_groups_delete
          ON groups FOR DELETE
            AS BEGIN
            UPDATE tc_groups SET groupid = NULL WHERE groupid IN (SELECT deleted.id FROM deleted)
          END
```

```sql
          CREATE TRIGGER tg_users_delete
          ON users FOR DELETE
            AS BEGIN
            DELETE FROM tc_user_user WHERE manageduserid IN (SELECT deleted.id FROM deleted)
          END
```

**Comment by tananaev** [comment]
What is the reason?

**Comment by Abyss777** [comment]
I have totally forget about MS SQL triggers when did renaming tables.

**Comment by tananaev** [comment]
Can you add it to your new changelogs? By the way, how's progress with it?

**Comment by SoheylAkhlaghiAmiri** [comment]
Thank you Abyss777;

it works for me.

**Comment by Abyss777** [comment]
Forget to mark it fixed in #4100
